### PR TITLE
Replace old repo URL with GitHub repo URL

### PR DESCRIPTION
**Pull request by carpetmaker**
# Replace old repo URL with GitHub repo URL

## Summary

Pretty much self-explanatory.

## Description

Replace the old project repo URL with the _GitHub_ repo URL in the following files:
* `lib.rs`
* `README.md`
  * update MIT license URL
  * update CI pipeline badge

### CODE REVIEW

**Comment by carpetmaker** [comment]
@just-a-stream pls don't merge after approval, just acknowledge it and then we'll chat about the best / optimal merge strategy offered by _GitHub_ PRs (_**Create a merge commit**_ vs _**Squash and merge**_ vs _**Rebase and merge**_).
I personally recommend _**Rebase and merge**_ for these tiny change PRs...
And maybe _**Squash and merge**_ for larger PRs, given that they have a proper commit message with body written... Alternatively those could be merged with a default merge commit. That would have an advantage of atomic commits & nice incremental overview of the changes applied, but creates an additional potentially-redundant commit. Also, in that case no merge commits inside the _feature_ branch itself should be allowed (outdated changes should be rebased insted).

**Comment by just-a-stream** [comment]
The change LGTM

**Review by just-a-stream — APPROVED** [review]
lgtm

**Pull request merged**
